samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/ItemContentsBean.java

```diff
@@ -88,6 +88,8 @@ public class ItemContentsBean implements Serializable {
 
 	private String feedbackValue;
 
+	private String infeedbackValue;
```

Comment:
please be verbose and don't try to save characters: incorrectFeedbackValue

Reply:
Changed to incorrectFeedbackValue